packages/flutter/lib/src/rendering/wrap.dart

```diff
@@ -235,6 +263,116 @@ class RenderWrap extends RenderBox with ContainerRenderObjectMixin<RenderBox, Wr
     markNeedsLayout();
   }
 
+  /// Determines the order to lay children out horizontally and how to interpret
+  /// `start` and `end` in the horizontal direction.
+  ///
+  /// If the [direction] is [Axis.horizontal], this controls which order
+  /// children are painted in (left-to-right or right-to-left), and the meaning
```

Comment:
just painted, or laid out as well?

Reply:
Laid out as well.  I'll update the flex docs too, which is where this text comes from.